horace_core/spaghetti_plot.m

```diff
@@ -362,7 +379,7 @@
 
 % Plot dispersion
 %----------------
-if nargout<1 || (~present.noplot)
+if nargout<1
     plot_dispersion(arrayfun(@d2d, wdisp),opt)
 end
```

Comment:
If I correctly understand the change, I would disagree with this. 
You can request plot and ask to return objects -- pretty reasonable situation. Here you allow one or anoter

Reply:
The change in this PR does not affect that at all, this is the original behaviour. 

The idea, I believe, is that if you return the object you clearly have an intent to transform it or use it elsewhere and so don't need to plot it immediately. 

This is indeed an overloaded use of the function and I am not a fan of that in general, but I recommend keeping the behaviour in this PR and if you want to discuss changing it, I would say open an issue.